admin_guide/allocating_node_resources.adoc

```diff
@@ -101,6 +101,50 @@ status:
 ----
 ====
 
+[[system-resources-reported-by-kubelet]]
+== System resources reported by kubelet
+
+Since openshift 3.3 the kubelet reports system resources allocated by container runtime and kubelet itself.
```

Comment:
I think there is a token for a product and version token, maybe a doc person can help give a pointer.

how about: "kubelet reports system resources utilized by the container runtime and kubelet"?